src/readerme.f

```diff
@@ -12,42 +14,37 @@
-C> [values-reading subroutines](@ref hierarchy)), just
-C> like if readmg() had been called instead.
+C> [values-reading subroutines](@ref hierarchy)).
 C>
-C> When using this subroutine, it's still necessary for the
+C> When using this subroutine, it's necessary for the
 C> application program to have previously called subroutine openbf() in
 C> order to associate a DX BUFR tables file with the message that is
-C> being input via MESG, and it's still also necessary to pass in the
+C> being input via MESG; it's also necessary to pass in the
 C> relevant LUNIT value as a call argument, even though in this case
 C> the subroutine will not actually try to read from the associated
 C> Fortran logical unit.
 C>
 C> If MESG contains a DX BUFR table message, the subroutine will
 C> store the contents internally and use them to process any
-C> future BUFR messages associated with LUNIT.  In this case, the
+C> future BUFR messages associated with LUNIT. In this case, the
 C> subroutine will return with IRET = 11, and any number of
 C> DX BUFR table messages passed in via consecutive calls to this
 C> subroutine will accumulate internally and be treated as a single DX
 C> BUFR table, up until a call is made where MESG no longer contains a
 C> DX BUFR table message.
 C>
-C> @param[in] MESG    -- integer(*): BUFR message
-C> @param[in] LUNIT   -- integer: Fortran logical unit number for
-C>                       BUFR file
-C> @param[out] SUBSET  -- character*8: Table A mnemonic for type of BUFR
-C>                        message that was read
-C>                        (see [DX BUFR Tables](@ref dfbftab)
-C>                        for further information about Table A mnemonics)
-C> @param[out] JDATE   -- integer: Date-time stored within Section 1 of
-C>                        BUFR message that was read, in format of either
-C>                        YYMMDDHH or YYYYMMDDHH, depending on the most
-C>                        recent call to subroutine datelen()
-C> @param[out] IRET    -- integer: return code
-C>                           - 0 = MESG was successfully read
-C>                           - 11 = MESG contained a DX BUFR table message
-C>                           - -1 = MESG contained an unrecognized
-C>                                  Table A message type
+C> @param[in] MESG - integer(*): BUFR message.
+C> @param[in] LUNIT - integer: Fortran logical unit number for BUFR file.
+C> @param[out] SUBSET - character*8: Table A mnemonic for type of BUFR
+C> message that was read (see [DX BUFR Tables](@ref dfbftab)
+C> for further information about Table A mnemonics).
+C> @param[out] JDATE - integer: Date-time stored within Section 1 of
+C> BUFR message that was read, in format of either YYMMDDHH or YYYYMMDDHH,
+C> depending on the most recent call to datelen().
+C> @param[out] IRET - integer: return code:
+C> - 0 MESG was successfully read.
+C> - 11 MESG contained a DX BUFR table message.
+C> - -1 MESG contained an unrecognized Table A message type.
```

Comment:
Once again for the record, IMO the removal of all of this whitespace makes the docblock a lot less readable.  Not everyone relies on just the rendered html from Doxygen for documentation - some of us still occasionaly edit the actual library routines and read it there.